docs/dev/libvirt-howto.md

```diff
@@ -286,6 +286,32 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console is depend on the route which is created by auth operator.
+To make it work we need to use the multi-stage installation and edit the `domain` for ingress.
```

Comment:
Suggest that you avoid 'the multi-stage installation' since that could be understood as indicating that the install command itself is to be run in stages where you run part of the install, do something, and then run the rest of the install.

Reply:
right, I had the same comment. multi-stage would likely refer here to the way the installer works... but this is actually abstracted away for most people, as they likely will experiment first with just `create cluster`. I would suggest to be more explicit and mention the order of the commands... or else mention how this could work AFTER the installation has been performed (if possible).

Reply:
Did some rewording, hope that make it bit clear.